scripts/setup.sh

```diff
@@ -15,6 +15,7 @@ fi
 : "${MAX_EXECUTORS:=5}"              # Default dynamic allocation executors to 5 if not set
 : "${EXECUTOR_CORES:=2}"             # Default executor cores to 2 if not set
 : "${MAX_CORES_PER_APPLICATION:=10}" # Default maximum cores per application to 10 if not set
+: "${DATANUCLEUS_AUTO_CREATE_TABLES:=true}" # Default DataNucleus auto create tables to true if not set
```

Comment:
In cdm-jupyterhub, this was enabled (true) for all users because we rely on JupyterHub groups to manage read/write permissions. To avoid future issues, I think it's better to keep it set to true by default.